pages/apis/graphql/cookbooks/jobs.md

```diff
@@ -169,3 +169,28 @@ mutation CancelJob {
   }
 }
 ```
+
+## Get retry information for a job
+
+To get information about how a job was retried (`retryType`), who retried the job (`retriedBy`) and which job was source of the retry (`uuid`).
```

Comment:
Thanks for this @tomowatt , minor stylistic suggestion to bring this in line with some of the other instructions on this page.

```suggestion
Gets information about how a job was retried (`retryType`), who retried the job (`retriedBy`) and which job was source of the retry (`uuid`).
```

Reply:
Cheers for spotting that @gilesgas, fixed it now!